io.github.quodlibet.QuodLibet.yaml

```diff
@@ -52,6 +58,67 @@ modules:
   - python-modules.json
   - kakasi.yaml
   - flathub-shared-modules/libappindicator/libappindicator-gtk3-introspection-12.10.json
+  - name: gst-libav
```

Comment:
I suggest using `meson`

Reply:
last time I looked it didn't handle the embedded libav case, but I'll give it a try, thanks :)

Reply:
Oh, I'm not a fan of the embedded libav case anyway, you should build what you want not generic defaults.

Reply:
gst-libav still doesn't support any build options. @TingPing do you mean I should include my own ffmpeg and build gst-libav against that?

Reply:
Yes, I do believe the end result would be better if you manually built `ffmpeg` since you don't need a lot of what `gst-libav` includes such as video codecs.

Reply:
> Yes, I do believe the end result would be better if you manually built ffmpeg since you don't need a lot of what gst-libav includes such as video codecs.

ok, thanks. I've opened #2 and will address this in the next update if that's ok with you.

Reply:
> since you don't need a lot of what gst-libav includes such as video codecs.

btw, it's still nice to be able to play the audio part of downloaded youtube videos :)

Reply:
You can do that without an h264 codec. Anyway yes all of your comments about improving things later are reasonable.

Reply:
> You can do that without an h264/vp9 codec.

Last time I tried gstreamer threw an error for missing codecs and stopped. I'll ask in #gstreamer

Thanks for your feedback